tests1/myfile1.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+echo $_POST[ 'phpcs should catch this problem' ];
+
+
+
+
+echo 'add one line';
+
+
+
+
+
+
+
+
+echo 'linter will catch this fatal;
```

Comment:
:no_entry_sign: **Error**: PHP syntax error: syntax error, unexpected ''linter will catch this fatal;' (T_ENCAPSED_AND_WHITESPACE) (*Generic.PHP.Syntax.PHPSyntax*).